pkg/kubelet/dockertools/manager.go

```diff
@@ -47,6 +47,7 @@ import (
 	"k8s.io/kubernetes/pkg/kubelet/prober"
 	"k8s.io/kubernetes/pkg/kubelet/qos"
 	kubeletTypes "k8s.io/kubernetes/pkg/kubelet/types"
+	kubetypes "k8s.io/kubernetes/pkg/kubelet/types"
```

Comment:
Master doesn't have a dup import. The initial alias used here is kubeletTypes, which doesn't match the naming convention for go package. We fixed it later. I believe the 1.1 was cut between, and such cleanup code wasn't cherry-picked. 

Should we fix this in 1.1 branch by cherry-pick the cleanup code too?